src/nvim/quickfix.c

```diff
@@ -3739,7 +3738,6 @@ void ex_copen(exarg_T *eap)
   incr_quickfix_busy();
 
   if (eap->addr_count != 0) {
-    assert(eap->line2 <= INT_MAX);
     height = (int)eap->line2;
```

Comment:
```
#if SIZEOF_INTMAX_T < SIZEOF_INT32_T
  assert(eap->line2 <= INT_MAX);
#endif
  height = (int)eap->line2;
```

Reply:
tbh I think we could just tell coverity to ignore this kind of thing.

    assert(eap->line2 <= INT_MAX); // coverity[...]

Reply:
IIRC `INT_MAX` is the maximum value of `int`, or `intmax_t`.